### DISCUSSION TITLE
Feature — Helm Charts

### DISCUSSION THREAD

**Discussion by itsezc** [cloudflare/workerd#1562] [Ideas]
A Helm Chart for `workerd` to run it on Kubernetes clusters easily with some documentation, would make large scale deployments far easier to approach.

**Comment by kentonv** [comment]
This sounds useful, but the Cloudflare Workers team may not be the right people to create this as we don't actually use Kubernetes to run workerd and we don't have much of a grasp on how it works. Perhaps some kubernetes expert out there could propose something?
